core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -1153,8 +1153,8 @@ class HttpGoogleServicesDAO(
   }
 
   private def getCloudResourceManagerWithCloudResourceManagerServiceAccount = {
```

Comment:
Can you change the name on this to `getCloudResourceManagerWithBillingServiceAccountCredential`?

Also, does this mean that the `cloudResourceManagerServiceAccountCredential` is unused? If so, should that instead move to using the right service account for that cred? It feels a bit like a code-smell to me to use the billingServiceAccountCredential here.